[Setting] PR이 머지 될 때 관련된 이슈 닫기

## 🚀 설명

main 브랜치 이외의 back, front 브랜치로 머지되는 PR 에 대해서도 본문에 포함된 이슈가 자동으로 닫힐 수 있게 합니다.

## 📖 할 일

- [x] github Actions를 사용한 처리

## 🔗 참고 자료

https://github.com/cli/cli/issues/2922
https://github.com/cli/cli/issues/8013

## 🎸 기타

코드블럭을 위한 백쿼트(```)가 있을 때 Actions에 오류가 있어 수정해야합니다.

- 관련 오류 메시지
```
Run # PR 본문에서 이슈 번호 추출
  # PR 본문에서 이슈 번호 추출
  ISSUE_NUMBERS=$(echo "## 🔢 관련 이슈
  
  #[2](https://github.com/Masidao/sip-of-travel/actions/runs/11340382251/job/31536712187#step:4:2)1 
  
  ## 🔧 변경 내용
  
  - cities 테이블 스키마 변경
    도시 테이블에 도시 대표 이미지 경로, 지도 축척-표현을 위한 level을 추가했습니다.
  
  - 도시 정보 반환형태 변경
     ```JSON
    {

.....



  shell: /usr/bin/bash -e {0}
  env:
    GITHUB_TOKEN: ***
/home/runner/work/_temp/[6](https://github.com/Masidao/sip-of-travel/actions/runs/11340382251/job/31536712187#step:4:6)23dcb93-3e3b-4391-8cfd-9dc0bd[7](https://github.com/Masidao/sip-of-travel/actions/runs/11340382251/job/31536712187#step:4:7)738b0.sh: line 12: unexpected EOF while looking for matching ``'
Error: Process completed with exit code 2.

--- a/.github/workflows/close-issues.yml
+++ b/.github/workflows/close-issues.yml
@@ -26,9 +26,10 @@ jobs:
       - name: 관련 이슈 닫기
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PR_BODY: ${{ github.event.pull_request.body }}
         run: |
           # PR 본문에서 이슈 번호 추출
-          ISSUE_NUMBERS=$(echo "${{ github.event.pull_request.body }}" | grep -oE '#[0-9]+' | sed 's/#//g')
+          ISSUE_NUMBERS=$(echo "${PR_BODY}" | grep -oE '#[0-9]+' | sed 's/#//g')
           
           # 이슈 번호가 하나라도 있는지 확인
           if [ -z "$ISSUE_NUMBERS" ]; then

